### PR TITLE
Added ocr option to close OCR window after opening browser for translation

### PR DESCRIPTION
**Pull request by gregorygregio**
Created an option in the Task Settings Form to close the OCR window when opening browser for translation
PZSitxowft

Fixes: #6385

### CODE REVIEW

File: ShareX/Forms/TaskSettingsForm.cs [hunk 1]
```diff
@@ -1365,6 +1366,10 @@ private void cbCaptureOCRAutoCopy_CheckedChanged(object sender, EventArgs e)
         {
             TaskSettings.CaptureSettings.OCROptions.AutoCopy = cbCaptureOCRAutoCopy.Checked;
         }
```
Comment (Jaex) [inline ShareX/Forms/TaskSettingsForm.cs:1368]: New line missing here.

File: ShareX/Tools/OCR/OCRForm.cs [hunk 2]
```diff
@@ -233,6 +233,10 @@ private void btnOpenServiceLink_Click(object sender, EventArgs e)
             if (!string.IsNullOrEmpty(Result) && cbServices.SelectedItem is ServiceLink serviceLink)
             {
                 serviceLink.OpenLink(Result);
+                if(Options.CloseWindowAfterOpeningServiceLink)
```
Comment (Jaex) [inline ShareX/Tools/OCR/OCRForm.cs:236]: There should be space after "if".

File: ShareX/Tools/OCR/OCRForm.cs [hunk 3]
```diff
@@ -222,7 +222,7 @@ private async void cbSingleLine_CheckedChanged(object sender, EventArgs e)
                 await OCR(bmpSource);
             }
         }
-
+
```
Comment (Jaex) [inline ShareX/Tools/OCR/OCRForm.cs:225]: ?

Comment (gregorygregio) [reply ShareX/Tools/OCR/OCRForm.cs:225]: I don't know where that came from

**Comment by Jaex** [comment]
Thanks!

**Pull request merged**